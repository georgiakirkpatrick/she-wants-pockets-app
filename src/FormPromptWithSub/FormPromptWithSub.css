.FormPromptWithSub {
    display: block;
}

.FormPromptWithSub_prompt {
    font-weight: bold;
    padding: 0 0 10px;
}

.FormPromptWithSub_subtitle {
    font-weight: 600;
    color: rgb(88, 88, 88)
}