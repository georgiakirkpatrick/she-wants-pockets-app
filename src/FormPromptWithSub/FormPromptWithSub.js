import React from 'react'
import './FormPromptWithSub.css'

const FormPromptWithSub = props => {
    return (
        <div className='FormPromptWithSub'>
            <p className='FormPromptWithSub_prompt'>
                {props.prompt}
            </p>
            <p className='FormPromptWithSub_subtitle'>
                {props.promptSubtitle}
            </p>
        </div>
    )
}

export default FormPromptWithSub