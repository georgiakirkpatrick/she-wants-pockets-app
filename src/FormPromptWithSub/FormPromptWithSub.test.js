import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import FormPromptWithSub from './FormPromptWithSub';

describe('FormPromptWithSub', () => {
        it ('renders without crashing', () => {
            const div = document.createElement('div');
            ReactDOM.render(<FormPromptWithSub />, div);
            ReactDOM.unmountComponentAtNode(div);
        });
    
        it ('render the UI as expected', () => {
            const tree = renderer
                .create(<FormPromptWithSub />)
                .toJSON();
            expect(tree).toMatchSnapshot()
        });
    });