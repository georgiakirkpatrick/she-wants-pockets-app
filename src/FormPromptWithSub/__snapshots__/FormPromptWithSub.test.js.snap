// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormPromptWithSub render the UI as expected 1`] = `
Array [
  <p
    className="FormPromptWithSub_prompt"
  />,
  <p />,
]
`;
